src/chains/ethereum/transaction/src/params.ts

```diff
@@ -56,7 +58,14 @@ export const Params = {
   /**
    * Fee for creation a transaction
    */
-  TRANSACTION_CREATION: 32000n,
+  TRANSACTION_CREATION_GAS: 53000n,
```

Comment:
I was a little confused by this, cos all the documentation says "Creation of a contract costs 32,000 gas", but turns out it's + base tx fee.

I feel that is worth a comment.